pkg/vault/client.go

```diff
@@ -281,9 +281,26 @@ func (v Client) RevokeToken() error {
 // MustRevokeToken same as RevokeToken but the programm is terminated with an error if this fails.
 // Should be used in defer statements only.
 func (v Client) MustRevokeToken() {
```

Comment:
Must is a quite strong word so it is a bit surprising that according to the comment it only "should" do some thing under certain conditions. Maybe call the function RevokeServiceToken instead?